website/client/src/components/shops/market/filter.vue

```diff
@@ -1,52 +1,52 @@
 <template>
-  <div class="form">
-    <h2 v-once>
-      {{ $t('filter') }}
-    </h2>
-    <div class="form-group">
-      <checkbox
-        v-for="viewOptionKey in Object.keys(viewOptions)"
-        :id="`category-${viewOptionKey}`"
-        :key="viewOptionKey"
-        :checked.sync="viewOptions[viewOptionKey].selected"
-        :text="viewOptions[viewOptionKey].text"
+  <filter-sidebar>
+    <filter-group>
+      <checkbox :checked.sync="viewOptions[viewOptionKey].selected"
+                :id="`category-${viewOptionKey}`"
+                :key="viewOptionKey"
+                :text="viewOptions[viewOptionKey].text"
+                v-for="viewOptionKey in Object.keys(viewOptions)"
```

Comment:
I know this isn't a change from this PR, but can we change it not to use `Object.keys`? v-for can be used to iterate over objects natively

Reply:
yep will do